src/open_producten/locaties/tests/api/test_contact_filters.py

```diff
@@ -0,0 +1,79 @@
+from uuid import uuid4
+
+from django.urls import reverse
+
+from rest_framework import status
+
+from open_producten.locaties.tests.factories import ContactFactory
+from open_producten.utils.tests.cases import BaseApiTestCase
+
+
+class TestContactFilters(BaseApiTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.path = reverse("contact-list")
```

Comment:
You can put this outside of `setUp()`, because `setUp()` is executed for each test, but the `url` is the same, and use `reverse_lazy` is more efficient
```suggestion
    path = reverse("contact-list")
```

Reply:
This for each test would be better